ceno_rt/riscv32im-ceno-zkvm-elf.json

```diff
@@ -0,0 +1,32 @@
+{
+  "arch": "riscv32",
+  "atomic-cas": true,
+  "cpu": "generic-rv32",
+  "crt-objects-fallback": "false",
+  "data-layout": "e-m:e-p:32:32-i64:64-n32-S128",
+  "disable-redzone": false,
```

Comment:
I'd love to add comments to explain the entries, but `json` doesn't let me.  Let me think of a way to overcome that.